elections/2021/governance-committee-election.md

```diff
@@ -0,0 +1,159 @@
+nomination” period, people can be nominated or nominate themselves by
+submitting a Pull Request adding said candidate to the
+[governance-committee-candidates.md](https://github.com/open-telemetry/community/blob/master/elections/2021/governance-committee-candidates.md)
+file in the OpenTelemetry community repository. The template in that file includes
+the following columns:
+* Full name
+* GitHub alias
+* Company affiliation (if applicable)
+* Short bio and/or reasoning to join the Governance Committee (no more than a
+  short paragraph)
+* _Optional_: photo/picture of a nominee
+
+The Pull Request will not be merged until the candidate has confirmed their
+desire to be nominated (if not self-nominating) and ratification has occurred
+via PR comments.
+
+The email address of the nominee should also be sent to
+[cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io) –
+it will be kept private and used only for candidate communications as the
+election process proceeds.
+
+The Governance Committee or appointed people will contact every nominee
+directly to ensure the commitment and desire to be nominated.
+
+# Voter Eligibility
+
+All [members of standing](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing)
+will automatically be eligible to vote. To confirm your eligibility status,
+see the [election announcements issue](https://github.com/open-telemetry/community/issues/852). If your code contributions do not meet
+eligibility requirements, but you believe your non-code contributions
+should make you eligible to vote, you can request an exemption by submitting
+an exemption request form [here](https://docs.google.com/forms/d/1TncjWYD6i43jPOZNfVl3_HlgVMUXZBlbvwxH0VVgMns).
+
+One of two options will be available in the form to prove eligibility:
+
+* GitHub handle
+* List either…
+  * Your contributions to OpenTelemetry that make you eligible to vote, or
+  * Your non-code contributions per the [Members of Standing section of the charter](https://github.com/open-telemetry/community/blob/master/governance-charter.md#members-of-standing):
+    particularly, your relationship to a well-known project that’s taken a hard
+    dependency on OpenTelemetry, and/or your involvement with a well-known
+    organization’s adoption of OpenTelemetry as an end-user.
+
+All exemption forms are private. Only the members of a current governance
+committee and election committee will have access to this information. Lists
+will be discarded one month after the election.
+
+# Vote
+
+Everyone with voting rights may log into
+[Helios Voting](https://vote.heliosvoting.org/helios/elections/bbeab1b8-1406-11ec-b980-eacead3a3158/view)
+using their GitHub account.
+Voting will be approval voting. Each voter may select up to 5 candidates.
+The 5 candidates with the most votes win the election.
+
+
+Late registrations to vote and requests to re-send the voting link will be
```

Comment:
is resending a link still actual?

Reply:
nope, you just have to sign in with github. but if someone doesn't have a github account, we can accommodate sending them a username/password.